mllib/src/main/scala/org/apache/spark/mllib/tree/RandomForest.scala

```diff
@@ -244,7 +245,10 @@ private class RandomForest (
 
     // Delete any remaining checkpoints used for node Id cache.
     if (nodeIdCache.nonEmpty) {
-      nodeIdCache.get.deleteAllCheckpoints()
+      Try(nodeIdCache.get.deleteAllCheckpoints()) match {
+        case Failure(e) => logWarning(s"delete all chackpoints failed. Error reason: ${e.getMessage}")
```

Comment:
@mengxr 
Thank you :exclamation: and I see that.
So, change and push the code.